examples/gpt-2/gpt2_generate_main.py

```diff
@@ -195,13 +205,16 @@ def run_model():
                     print('Input should not be empty!')
                     raw_text = input("Model input >>> ")
             except EOFError:
+                print("Input not supported.")
```

Comment:
The message is not accurate. `EORError` is raised when the user sends an `EOF` character as input, in Unix-based systems this is `Ctrl+D`. This indicates that the users wants to quit the interactive environment, so a response of `"Input not supported."` would be confusing.

Personally I don't think a message is needed here, for instance `python` does not print anything on EOF. Other interactive envs like `ipython` print a confirmation message `"Do you really want to exit ([y]/n)?"`, but I don't think it's necessary here. If we really want a message, let's just say something along the lines of `"EOF entered, quitting."`.